xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.cpp

```diff
@@ -419,7 +419,9 @@ bool CRenderManager::Flush(bool wait, bool saveBuffers)
   {
     CLog::Log(LOGDEBUG, "{} - flushing renderer", __FUNCTION__);
 
+#ifndef TARGET_WINDOWS // this may cause deadlock on Windows
     CSingleExit exitlock(CServiceBroker::GetWinSystem()->GetGfxContext());
```

Comment:
Is this causes deadlock or fixes it?

Reply:
Line `CSingleExit exitlock(CServiceBroker::GetWinSystem()->GetGfxContext());` causes deadlock with Nvidia in combination with sync playback to display setting enabled. Intel (NUC) seems unaffected and works fine with/without this line.

On others platforms I'm not sure/not tested so removed it only for `TARGET_WINDOWS`

Reply:
please change the comment, because it's not obvious.